### PR TITLE
Add link entities command

### PR DESCRIPTION
**Pull request by Morb0**
## About the PR

Adds command to link entities using `LinkedEntityComponent`, for example for then adding `PortalComponent`

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Content.Server/Teleportation/LinkEntitiesCommand.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
+using Content.Server.Administration;
+using Content.Shared.Administration;
+using Content.Shared.Teleportation.Systems;
+using Robust.Shared.Console;
+
+namespace Content.Server._Stalker.Teleportation;
+
+[AdminCommand(AdminFlags.Admin)]
+public sealed class LinkEntitiesCommand : IConsoleCommand
```
Comment (AJCM-git) [inline Content.Server/Teleportation/LinkEntitiesCommand.cs:9]: Use ToolsheCommand (ping moony if u got questions)

**Review by AJCM-git — CHANGES_REQUESTED** [review]

File: Content.Server/Teleportation/LinkEntitiesCommand.cs (hunk 1 above)
Comment (Morb0) [reply Content.Server/Teleportation/LinkEntitiesCommand.cs:9]: it complicated and hard :despair:

**Review by moonheart08 — CHANGES_REQUESTED** [review]
Use ToolshedCommand. 
It's not hard, look at an existing command for ref.

**Comment by metalgearsloth** [comment]
Re-open if you come back to this.

**Pull request closed without merging**